docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -622,7 +622,7 @@ part of the server.
 
 When using Vault as a backend you can share configuration with
 all applications by placing configuration in
-`{backend}/application`.  For example, if you run this Vault command
+`secret/application`.  For example, if you run this Vault command
```

Comment:
I can imagine that was probably the idea, but everywhere else in the file is hardcoded to `secret`. Which is the default value anyway and used by all the examples.